docs/core/tools/dotnet-test.md

```diff
@@ -496,6 +499,9 @@ With Microsoft Testing Platform, `dotnet test` operates faster than with VSTest.
 
   Short form `-r` available starting in .NET SDK 7.
 
+> [!NOTE]
+> Running tests for a solution with a specific `Runtime Identifier` is not supported. If needed, this should be specified at the individual project level instead.
```

Comment:
This needs to be indented. I would also remove the space in "Runtime Identifier"